overlord/overlord.go

```diff
@@ -117,12 +118,31 @@ func (o *Overlord) ensureTimerSetup() {
 	o.ensureNext = time.Now().Add(ensureInterval)
 }
 
-func (o *Overlord) ensureTimerReset() {
+func (o *Overlord) syncEnsurePrep() time.Time {
```

Comment:
The original method name feels much better. syncEnsurePrep gives no hints about what it actually does.